docs/reviewer-guidelines.md

```diff
@@ -169,11 +169,34 @@ If grub is used:
    filesystem modules have patchy security history and are best left
    disabled.
 
+## systemd-boot
+
+Does the submitter use systemd-boot as a bootloader? This is also used
+in certain distributions, but less common than grub.
+
+If systemd-boot is used:
+
+1. Is it used exclusively, or provided alongside grub as an alternative
+   package?
+1. Is it intended to be used with BLS
```

Comment:
numbering is wrong

Reply:
Check the rendered version, markdown does the counting when displaying, and shows the right number. This is how all other existing numbered lists are handled in this document.

Reply:
ah, was not aware. Thanks